src/pages/reset-password/[resettoken].tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+import { useApiClient } from '@common/contexts';
+
+import { HStack, VStack } from '@chakra-ui/react';
+import { AppContainer, Logo } from '@src/common/components';
+import { ResetPasswordForm } from '@src/modules/auth';
+import { AuthRoute } from '@src/constants/PageRoutes';
+import { useResetPasswordMutation } from '@generated/graphql.queries';
+import { isNil } from 'lodash';
+
+export default function ForgetPassword() {
+  const { t } = useTranslation('auth');
+  const { gqlClient } = useApiClient();
+  const head = { title: t('reset-password.title') };
+
+  const [resetToken, setResetToken] = useState('');
+  const [successMessage, setSuccessMessage] = useState('');
+  const [successAction, setSuccessAction] = useState('');
+  const [successActionLink, setSuccessActionLink] = useState('');
+
+  const token = useRouter().query.resettoken as string;
```

Comment:
You can actually deconstruct the `resetToken` like this.

```ts
const {query: {resetToken}} = useRouter().
```

and it's `resetToken` not `resettoken`

Reply:
this has been updated